### ISSUE TITLE
TPA not working

### ISSUE THREAD

**Issue by Gayytis** [Zrips/CMI#4983]
**Description of issue:**
A video is a better, [click here to see the issue](http://thomasclosson.fr/videos/tpaissue.mp4)
It's an issue with the /tpa and /tpaccept command. The player is not teleported to the destination

---

**CONFIG SECTION (DELETE IF NOT RELEVANT):**

![image](https://user-images.githubusercontent.com/65625456/105762858-463a0b00-5f55-11eb-865b-64d51df4ba36.png)

---

**Cmi Version (using`/cmi version`):** 8.8.0.7

**Server Type (Spigot/Paperspigot/etc):** PaperSpigot

**Server Version (using `/ver`):** 443

**Comment by CloudeLecaw** [comment]
You player have a 120secounds cooldown u seem to breake.. 
You have defines a Warm-up?

**Comment by x1p** [comment]
Same problem here.
The tpa request can be accepted normally, but it won't teleport the player.

**Comment by Gayytis** [comment]
> You player have a 120secounds cooldown u seem to breake..
> You have defines a Warm-up?

I removed modules ^^

> Edit: Problem found: Happens when the warmup module is disabled in Modules.yml in combination with Teleport.TpaWarmup >0

Imma change it so

**Comment by strahil211** [comment]
Also having this problem

**Comment by mrfloris** [comment]
Why set a tpa warmup and then disable the module? Maybe I misunderstand, but that makes sense to me why it then wouldn't work. 

Disable the warmup or the tpa, or enable the required module so warmups are handled?

**Comment by mrfloris** [comment]
Back after some testing.

I got paper 1.16.5 and latest spigotmc build of cmi, and run a fresh new server without any other plugins.

Out of the box /cmi tpa player works, they can accept it and get teleported. 

i stopped the server and disabled the warmup module, again, still haven't touched the plugins/cmi/ yml files, didn't configure anything besides modules.yml. > warmup > false

started the server, and i can run the commands, but the player wont get teleported. So I guess that confirms it's related to turning off warmup. 

If you have an issue on the server and it has an impact on players playing, then turn on warmup module for now until zrips has looked into it.

Side note, i saw mention of `TpaWarmup: 0` being bigger than 0, but this has no impact. I stopped the server, left warmup module to false, and set TpaWarmup from 3 to 0, and started the server. Players will still be unable to teleport.

Enabling the warmup module fixed it right away. 

I hope this additional information is helpful 

<img width="1392" alt="Screenshot 2021-01-27 at 05 30 08" src="https://user-images.githubusercontent.com/28841349/105943054-bd0ffa80-6060-11eb-9b95-856d69276855.png">
